extension/src/test/LanguageFunctions.test.ts

```diff
@@ -1236,6 +1241,80 @@ suite("Language Functions Tests", function () {
       "Expected all targets to have the state set to needsReviewTranslation."
     );
   });
+
+  test("importDtsTranslatedFile: Error", function () {
+    const settings = SettingsLoader.getSettings();
+    const dtsZipPath = path.join(
+      __dirname,
+      testResourcesPath,
+      "import-dts-test.zip"
+    );
+    const langXliffArr: Xliff[] = [
+      Xliff.fromString(ALObjectTestLibrary.getXlfHasNABTokens()),
+    ];
+    langXliffArr[0].targetLanguage = "no-PE";
+    const expectedErrMsg = `There are no xlf file with target-language "sv-SE" in the translation folder (${settings.translationFolderPath}).`;
```

Comment:
The error message thrown is a bit misleading.

Reply:
I was just thinking about that... shouldn't it be "no-PE" in the error message?

Reply:
We could output all the target languages in `langXliffArr` and have the error message say; `Found no xlf files matching target languages xx-XX, no-PE in <path-to-dts-zip>`

Reply:
Aaah, now I understand (had to peek into the source of this error message=
"sv-SE" comes from the zip file.

Maybe that should be added to the error message? Something line "There are no xlf file with target-language "sv-SE" (found in zip file) in the translation folder (${settings.translationFolderPath})." or similar?

Reply:
Now I re-read you message and realizes that you where trying to write the same, but from another point of view. I think your suggestion is better, but since I didn't understand it properly we might clarify it a bit more... :)

Maybe something like this?
`Found no xlf files matching target languages xx-XX that was found in <path-to-dts-zip>. Target languages in xlf files: <list of all target languages in the xlf files>`

Reply:
Done :)